Connectivity Widget

Steedie opened the pull request:
## What

- Shows connectivity chain when cps is <= 5
  - Orange when `cps` <= 5
  - Red when < 1

![image](https://github.com/user-attachments/assets/350cc699-d28d-457b-9b27-1e884c60fadc)

## Why

In cases where the player is unable to move because commits per second is 0, or extremely low, we want an on-screen indicator to inform them of this.

## Notes
The easiest way I've found to solo test this is to move a window to another monitor and `cps` should drop to 0 momentarily

Resolves: #119

ldunnplaymint commented:
As far as I can tell it works but I think the widget is too small and I think showing a "wifi" symbol might be a bit more universal in it's intent

Steedie commented:
> As far as I can tell it works but I think the widget is too small and I think showing a "wifi" symbol might be a bit more universal in it's intent

Updated:

![image](https://github.com/user-attachments/assets/faff27c3-b41c-4559-b69f-f6d2353c76be)

Also, fixed bug where it shows before game has started.

Commit: https://github.com/playmint/playerchain-demo/pull/144/commits/459e256547992d0dc3ccb2678e95a3cecfa3878f
